<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="LAPTOP-78I34NMD" failures="1" tests="1" name="tests.Solution1" time="55.931" errors="0" timestamp="2021-08-31T14:28:00 IST" skipped="0">
  <testcase classname="tests.Solution1" name="flipkartAction" time="55.931">
    <failure message="By.xpath: //div[contains(text(),&#039;APPLE MacBook Air Core i5 5th Gen - (8 GB/128 GB S&#039;)]Element not found expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: By.xpath: //div[contains(text(),'APPLE MacBook Air Core i5 5th Gen - (8 GB/128 GB S')]Element not found expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertTrue(Assert.java:45)
at utils.ActionUtil.waitForElementVisibility(ActionUtil.java:93)
at tests.Solution1.flipkartAction(Solution1.java:23)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:78)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
at java.base/java.lang.Thread.run(Thread.java:831)
]]>
    </failure>
  </testcase> <!-- flipkartAction -->
  <system-out/>
</testsuite> <!-- tests.Solution1 -->
